src/main/resources/file-download-context.xml

```diff
@@ -31,20 +31,21 @@
 		<constructor-arg name="password" index="3" value="${release.database.password}"/>
 		<constructor-arg name="port" index="4" value="${release.database.port}"/>
 	</bean>
-	
-	
+
+
 	<bean id="objectCache" class="org.reactome.addlinks.db.ReferenceObjectCache" scope="singleton">
 		<constructor-arg index="0" type="org.gk.persistence.MySQLAdaptor" name="adapter" ref="dbAdapter"/>
 		<constructor-arg index="1" type="boolean" name="lazyLoad" value="${lazyLoadCache}" />
 	</bean>
-	
+
 
 	<util:list id="fileRetrieverFilter" value-type="java.lang.String">
-		<value>UniProtToKEGG</value>
-		<value>KEGGRetriever</value>
+		<value>COSMIC</value>
+<!--		<value>IntEnz</value>
```

Comment:
Is the commenting out just for one run?  If so, should it be removed from the PR?